osu.Game/Graphics/UserInterface/OsuTabControl.cs

```diff
@@ -65,6 +84,15 @@ public Color4 AccentColour
             Masking = true
         };
 
+        protected override void UpdateAfterChildren()
+        {
+            base.UpdateAfterChildren();
+
+            // dont bother calculating if the strip is invisible
+            if (strip.Colour.MaxAlpha > 0)
+                strip.ResizeWidthTo(StripWidth(), 500, Easing.OutQuint);

```

Comment:
We previously found out doing per-frame transforms is kinda expensive. Use the interpolation functions directly instead of this:
`strip.Width = Interpolation.ValueAt(MathHelper.Clamp(Clock.ElapsedFrameTime, 0, 1000), strip.Width, StripWidth(), 0, 500, Easing.OutQuint);`